test_generator.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from pysize import get_size
+
+from generator import create_random_data_list, create_random_job_list
```

Comment:
It not easy, I think I need to create 2 folders and to add files to them.
Also, need `__init__.py`
Link for this case - https://stackoverflow.com/questions/41816973/modulenotfounderror-what-does-it-mean-main-is-not-a-package
Need to do the relative import?
Because many many people tell absolute import better.
But for learning I can try to do it. @webknjaz

Reply:
I can use `from .script import...` if add `__init__.py` to the folder.

Reply:
yes